README.md

```diff
@@ -2,83 +2,64 @@
 [![Build Status](https://travis-ci.org/terryma/vim-multiple-cursors.svg)](https://travis-ci.org/terryma/vim-multiple-cursors)
 [![Issue Stats](http://issuestats.com/github/terryma/vim-multiple-cursors/badge/pr?style=flat)](http://issuestats.com/github/terryma/vim-multiple-cursors)
 [![Issue Stats](http://issuestats.com/github/terryma/vim-multiple-cursors/badge/issue?style=flat)](http://issuestats.com/github/terryma/vim-multiple-cursors)
-## Contents
```

Comment:
I love Table of Contents and I'd keep it even if it's small.

Reply:
Yeah, I was kind of hesitant about this one.
The more I think about it, the more I think you're right.
Will add it back.